app/javascript/dashboard/components-next/message/chips/File.vue

```diff
@@ -53,11 +67,16 @@ const textColorClass = computed(() => {
 
 <template>
   <div
-    class="h-9 bg-n-alpha-white gap-2 items-center flex px-2 rounded-lg border border-n-container"
+    class="h-9 bg-n-alpha-white gap-2 overflow-hidden items-center flex px-2 rounded-lg border border-n-container"
   >
     <FileIcon class="flex-shrink-0" :file-type="fileType" />
-    <span class="mr-1 max-w-32 truncate" :class="textColorClass">
-      {{ fileName }}
+    <span
+      :title="fileName"
+      :class="textColorClass"
+      class="inline-flex items-center text-sm overflow-hidden flex-1 min-w-0"
+    >
+      <span class="truncate min-w-0 max-w-32">{{ fileNameWithoutExt }}</span>
+      <span class="flex-shrink-0 whitespace-nowrap">.{{ fileType }}</span>
```

Comment:
@iamsivin what if the file name is just `document` we won't have a `filteType`, in that case we will just render an empty `.` after the file name. 

Can you take a look?

Reply:
Done